lib/govuk_app_config/govuk_i18n.rb

```diff
@@ -0,0 +1,36 @@
+module GovukI18n
+  {
```

Comment:
I wonder if we should give this an explicit name such as

```rb
def plurals
  ...
end
```

or 

```rb
PLURALS = ...
```

Otherwise I don't think it'll be possible to reference this hash from another location.

Reply:
Yeah, I'll turn that into a method